File: Java/src/htmlwalker/tag/html/TagBig.java
package htmlwalker.tag.html;

import htmlwalker.exception.HtmlWalkerException;
import htmlwalker.platform.WalkerFactory;
import htmlwalker.tag.base.ContentContainerTag;
import htmlwalker.visitor.ITagGenericVisitor;
import htmlwalker.visitor.ITagVoidVisitor;

public class TagBig extends ContentContainerTag
{
    public TagBig(WalkerFactory factory) { super(factory, factory.createBig()); }

    public TagBig(WalkerFactory factory, Object tag) { super(factory, tag); }

    @Override
    public void visit(ITagVoidVisitor visitor) throws HtmlWalkerException { visitor.visit(this); }

    @Override
    public <T> T visit(ITagGenericVisitor<T> visitor) throws HtmlWalkerException { return visitor.visit(this); }
}

File: CSharp/HtmlWalker/Tag.Html/TagBig.cs
using HtmlWalker.Platform;
using HtmlWalker.Tag.Base;
using HtmlWalker.Visitor;

namespace HtmlWalker.Tag.Html
{
    public class TagBig : ContentContainerTag
    {
        public TagBig(WalkerPlatform.WalkerFactory factory) : base(factory, factory.CreateBig()) { }

        public TagBig(WalkerPlatform.WalkerFactory factory, object tag) : base(factory, tag) { }

        public override void Visit(ITagVoidVisitor visitor) { visitor.Visit(this); }

        public override T Visit<T>(ITagGenericVisitor<T> visitor) { return visitor.Visit(this); }
    }
}
